de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5533410_cli_gh5633_c_invoice_line_aloc_c_uom_id_update_nulls.sql

```diff
@@ -0,0 +1,5 @@
+update c_invoice_line_alloc ila set qtyinvoicedinuom=il.qtyentered, c_uom_id=il.c_uom_id
+from c_invoiceline il
+where il.c_invoiceline_id=ila.c_invoiceline_id
+and ila.c_uom_id is null;
```

Comment:
tried on a big DB with almost 4M `c_invoice_line_alloc`s; took 1m 30sec, so it's fine 👍